### DISCUSSION TITLE
New User 3.45.0 on Windows of Lossless Cut. Missing extension and strange playback progress start time

### DISCUSSION THREAD

**Discussion by pcumming** [mifi/lossless-cut#1225] [Support]
New User 3.45.0 on Windows of Lossless Cut. Missing extension and strange playback progress start time

Appreciate any help.
Opened a 2gb Mp4 movie
I then set the start and stop pointers using the hand. I picked a clip that was like 2 minutes and started at the 27 minute mark until 29 minute
I clicked export.

1. The Exported file did not end in .mp4  It just had no extension. So I had to rename it. 
2. When I played the clip in VLC or Media Player on Windows 10 the progress of the video bar immediately jumped to 27 minutes in both programs and then played the 2 minutes.
Thoughts?

Thanks so much

Trying to preserve some of my wife's family that is no longer around. 

Sincerely
Peter

**Comment by mifi** [comment]
hi.
1. have you changed the output filename template? if so you can try to reset it to the standard name
2. could u provide a report from the help menu. also maybe try to disable all tracks except video and see if that helps
